xla/pjrt/c/pjrt_c_api.h

```diff
@@ -1576,6 +1576,8 @@ struct PJRT_Executable_DeserializeAndLoad_Args {
   PJRT_Client* client;
   const char* serialized_executable;
   size_t serialized_executable_size;
+  const char* compile_options;
```

Comment:
Is this the same compile options proto as in Compile()? 

https://github.com/openxla/xla/blob/b46d80b8b651ca230d5b797a128dbd3dbe30de5c/xla/pjrt/c/pjrt_c_api.h#L2106

If so, mind adding a comment explicitly stating it's the same?

Reply:
Done.